docs/data/routes/docs/fundamentals/services/dictionary/sitecore-dictionary-service/en.md

```diff
@@ -0,0 +1,27 @@
+---
+name: sitecore-dictionary-service
+routeTemplate: ./data/component-templates/article.yml
+title: Sitecore Dictionary Service
+---
+
+# Sitecore Dictionary Service
+
+Sitecore Headless Services provides a REST endpoint for retrieving an app-specific translation dictionary, backed by a Sitecore feature called *Dictionary Domains*. 
+
+> Note: If you provide the `sc_lang` param to [Layout Service](/docs/fundamentals/services/layout/sitecore-layout-service) requests, the service will return content in the specified language. The Dictionary Service returns translated phrases in your app
+
+The endpoint format for this API is `/sitecore/api/jss/dictionary/<app>/<language>/`.
+
+For example, a request to `http://JssReactWeb.dev/sitecore/api/jss/dictionary/JssReactWeb/es-MX/` will return:
+
+```json
+{
+    "lang": "es-MX",
+    "app": "JssReactWeb",
+    "phrases": {
+        "Copyright": "{{year}} Inicio Bootstrap",
+        "Home": "Inicio",
+        "Start Bootstrap": "Inicio Bootstrap"
+    }
+}
+```
```

Comment:
I'm wondering if this content should be wrapped into the API Reference page... since it's describing the shape of the return object, which can be determined from the TS type.

Maybe the thing we're missing in the API Reference page is links to the underlying TS classes?

Reply:
Since this describes the data returned by the Headless Services Module, we decided to leave this alone and include a ref doc link to the front-end part of the docs when the ref docs are generated.